src/pages/HomePage/ContactSection.jsx

```diff
@@ -0,0 +1,148 @@
+                padding: "8px",
+                marginY: "28px",
+              }}
+            >
+              <IconButton aria-label="email" sx={{ color: "white" }}>
+                <EmailIcon />
+              </IconButton>
+              <Typography
+                variant="h3"
+                sx={{
+                  lineHeight: 1.6,
+                  marginLeft: "0.5rem",
+                }}
+              >
+                Email us
+                <br />
+                <Link
+                  href="mailto:help@thesneakersociety.com"
+                  sx={{ color: "white" }}
+                >
+                  help@thesneakersociety.com
+                </Link>
+              </Typography>
+            </Box>
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "row",
+                alignItems: "center",
+                textAlign: "left",
+                border: "1px solid white",
+                padding: "8px",
+                marginY: "8px",
+              }}
+            >
+              <IconButton aria-label="instagram" sx={{ color: "white" }}>
+                <FaInstagram />
+              </IconButton>
+              <Typography
+                variant="h3"
+                sx={{
+                  lineHeight: 1.6,
+                  marginLeft: "0.5rem",
+                }}
+              >
+                DM us on Instagram
+                <br />
+                <Link
+                  href="https://instagram.com/thesneakersociety1"
+                  sx={{ color: "white" }}
+                >
+                  @thesneakersociety1
+                </Link>
+              </Typography>
+            </Box>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <Box
+              sx={{
+                width: "100%",
+                overflow: "hidden",
+                paddingTop: "50%",
+                position: "relative",
+                borderRadius: "4px",
+              }}
+            >
+              <img
+                src="assets/SNEAKER SOCIETY (Transparency).png"
+                alt="Sneaker Society Logo"
+                style={{
+                  position: "absolute",
+                  top: 15,
+                  left: 0,
+                  width: "100%",
+                  height: "100%",
+                  objectFit: "cover",
+                  borderRadius: "4px",
+                }}
+              />
+            </Box>
+          </Grid>
+        </Grid>
+        <Stack
+          direction="row"
+          spacing={SPACING}
+          sx={{
+            justifyContent: "space-between",
+          }}
+        >
+          <Link href="/TOS" sx={{ textDecoration: "none", color: "white" }}>
+            Terms & Conditions
+          </Link>
+          <Typography variant="body2">
+            © 2023 The Sneaker Society - All rights reserved
+          </Typography>
+        </Stack>
```

Comment:
This should be a footer and suck at the bottom takwe a look at how it looks on mobile

Reply:
updated for mobile